Automattic/wp-calypso
# Update Newspack blocks to v1.37.1

### Changes proposed in this Pull Request
Bump newspack blocks to v1.37.1 and resolve peer dependency warnings via .yarnrc.yml. 

### Releasing to npm
Here's how I did the release to npm:
1. Cloned the newspack blocks repo
2. Checked out the commit tagged with the 1.37 release
3. Changed the "name" field in `package.json` to include `@automattic/` as a prefix
4. Removed the `private: true` field in package.json.
5. Ran the following commands:

```sh
npm ci
npm run clean
NODE_ENV=production npm run build

npm publish
```

### Testing instructions
1. Sync the build to your sandbox (`install-plugin.sh etk update/newspack-blocks-1.37`)
2. Load the editor in a sandboxed site, and verify the ETK version in the "more menu" dropdown matches the version on your sandbox.
3. Smoke test inserting and using the newspack blocks.
